server/app.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express')
+const mongoose = require('mongoose')
+const cors = require('cors')
+require('express-async-errors')
+
+// Require routers here //
+
+const { MONGODB_URI } = require('./utils/config')
+const { infoLogger, errorLogger } = require('./utils/logger')
+const middleware = require('./utils/middleware')
+
+const app = express()
+
+/* Database Connection */
+logger.info('connecting to', MONGODB_URI)
```

Comment:
`logger` is undefined. Since you have used the destructuring syntax, there is no logger object anymore. Please, implement this change.